src/ontology/cl-edit.owl

```diff
@@ -28077,15 +28077,14 @@ AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4030014 "epithelial cell of
 AnnotationAssertion(rdfs:label obo:CL_4030014 "kidney loop of Henle long descending thin limb inner medulla epithelial cell"^^xsd:string)
 SubClassOf(obo:CL_4030014 obo:CL_1001111)
 
-# Class: obo:CL_4030015 (collecting duct intercalated cell type A)
+# Class: obo:CL_4030015 (kidney collecting duct alpha-intercalated cell)
 
-AnnotationAssertion(Annotation(<http://purl.org/dc/terms/contributor> <https://orcid.org/0000-0003-2804-127X>) Annotation(oboInOwl:hasDbXref "PMID:31249312"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:9890303"^^xsd:string) obo:IAO_0000115 obo:CL_4030015 "Epithelial cell that is part of the collecting duct and secretes protons via an apical H+-ATPase and reabsorbs bicarbonate by a Cl-/HCO3-exchanger (AE1) located in the basolateral plasma membrane."^^xsd:string)
+AnnotationAssertion(Annotation(<http://purl.org/dc/terms/contributor> <https://orcid.org/0000-0003-2804-127X>) Annotation(oboInOwl:hasDbXref "PMID:31249312"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:9890303"^^xsd:string) obo:IAO_0000115 obo:CL_4030015 "A kidney collecting duct intercalated cell that secretes protons via an apical H+-ATPase and reabsorbs bicarbonate by a Cl-/HCO3-exchanger (AE1) located in the basolateral plasma membrane."^^xsd:string)
```

Comment:
similarly, this should follow the equivalent class. ("a [renal alpha-intercalated cell](http://purl.obolibrary.org/obo/CL_0005011) that is part of the [collecting duct of renal tubule](http://purl.obolibrary.org/obo/UBERON_0001232).") the rest of the information can go into the gloss 

See https://douroucouli.wordpress.com/2019/07/08/ontotip-write-simple-concise-clear-operational-textual-definitions/ "Genus-differentia definitions are your friend" and "Match Textual and Logical Definitions" section